docs/en/glossary.md

```diff
@@ -1,17 +1,107 @@
+generally dispense medications directly to patients.
+
+* **Dispensing Pharmacy**
+A Dispensing Pharmacy (Pharmacie D'Usage in French) is a pharmacy that
+dispenses stock to patients, wards, and services.  Normally, stock is not
+transferred from Dispensing Pharmacies to other Pharmacies.
+
+
+---
 ## I
 
-##### Integration
+##### INTEGRATION
 Integration is a method of increasing or decreasing the quantity in stock of a particular stock lot.  It's used to perform adjustments to the stock in a depot/pharmacy after an inventory is performed and quantities need to be adjusted to match the real quantity in stock.
 
 
+---
 ## S
 
-##### Service
-A service is a department in the hospital.  Services can be assigned invoices, consume medications, contain staff, and have both profit and loss centers.   They are useful for understanding where in the hospital financial activities are taking place.
+##### SERVICE
+
+A service is a department in the hospital such as "pediatrics",
+"ophthalmology", "internal medicine", "pharmacy", and "administration".
+Services can be assigned invoices, consume medications, contain staff, and
+have both profit and loss centers.  They are useful for understanding where in
+the hospital financial activities are taking place.
+
+#### STOCK
+"Stock" are any items used in the operation of a hospital or clinic.  Examples
+include drugs for treating patients, medical equipment and supplies, and
+administrative supplies.  "Stock" items are often purchased in "Lots" which
+include a number of the stock items.  Note that "Lot Numbers" are sometimes
+given by manufacturers and are not necessarily unique to BHIMA.
+
+#### STOCK ADJUSTMENT
+When a physical inventory of stock reveals that the actual count of stock
+items differs from the count in the "Theoretical Quantity" (as tracked by the
+BHIMA system), then an "Stock Adjustment" should be performed to update the
+BHIMA system (and the "Theoretical Quantity").
+
+#### STOCK ENTRY
+"Stock entries" occur when an item of stock is added to a Depot.  Types of
+stock entries include:
+
+* **Purchase** - stock items entering the Depot from a fulfilled purchase order
+
+* **Donation** - stock items entering the Depot from a donation
+
+* **Integration** - stock items entering the Depot that do not have a known
+  origin but are necessary to be tracked in the system.  This often happens
+  during start-up - stock is found on the shelf, but it isn't necessarily
+  clear where it came from.  It's easiest to simply "integrate" the stock
+  as-is, rather than try to reproduce the history exactly from previous
+  documentation.
+
+* **Transfer** - stock items entering the Depot due to a transfer from some
+  other Depot.
+
+#### STOCK EXIT
+"Stock exits" occur when an item is removed from a Depot.  This includes exits due to:
+
+* **Exit to an individual* - Stock items leaving the depot for use in the
+  hospital or clinic
```

Comment:
This definition isn't accurate.  The exit to an individual is stock items leaving the depot directly to a patient.   They can optionally be associated with an invoice billed to that patient.